tests/console/journalctl.pm

```diff
@@ -42,9 +42,15 @@ sub check_syslog {
     return !is_tumbleweed;
 }
 
+sub isPublicCloud {
+    return get_var('PUBLIC_CLOUD');
+}
+
 sub run {
     my ($self) = @_;
     $self->select_serial_terminal;
+    # TODO: Handle reboots on public cloud
+    record_info("publiccloud", "Public cloud detected - omitting reboot for now") if isPublicCloud;
```

Comment:
I think a comment on line 70 would be enough. No need for the record info.
But, if you think that the record info is needed, it would be better on an else clause on line 73.

Reply:
I think this is useful to have, as it reminds us to fix this for publiccloud, once a fix is possible (we need first to do some cleanup on the underlying ssh tunnels).

But indeed, it might be better as `else` clause. I update that accordingly and schedule a short test run on SLE15-SP1

Reply:
[SLE15-SP1](http://phoenix-openqa.qam.suse.de/tests/1417#) run passes with the new changes